### PR TITLE
Balder/remove cluster from system info

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth and @gjoranv PR

### CODE REVIEW

**Review by hakonhall — APPROVED** [review]
LGTM for main applications.  I don't know anything about public Vespa Cloud usage of SystemInfo.cluster().

File: hosted-zone-api/src/main/java/ai/vespa/cloud/SystemInfo.java [hunk 1]
```diff
@@ -42,8 +46,8 @@ public Cloud cloud() {
         return cloud;
     }
 
-    /** Returns the cluster this is part of */
-    public Cluster cluster() { return cluster; }
```
Comment (gjoranv) [inline hosted-zone-api/src/main/java/ai/vespa/cloud/SystemInfo.java:46]: This is used by stream-ranking:
https://git.ouryahoo.com/streambe/massmedia_serving/blob/master/homerun/src/main/java/com/yahoo/jarvis/docproc/CleanupExpiredGlobalscoresDocProc.java#L84

**Review by gjoranv — COMMENTED** [review]
Found one usage of ´SystemInfo.cluster()´. Cluster size for calculating the execution rate of a cleanup task.

**Comment by baldersheim** [comment]
Handled by #24756

**Pull request closed without merging**